.github/workflows/workflow.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - name: Lint
         run: |
           echo "Running isort"
-          isort -c -sp .
```

Comment:
There is no need to read the verbose messages and figure out what's wrong manually. Just run `./dev/linter.sh` with correct version of isort/black, and all isort/black issues should be fixed.

Reply:
@ppwwyyxx I am using `isort==4.3.21`, but the way `dev/linter.sh` fixes the code is not accepted by `isort -c -vb -sp .` at `workflow.yaml`

It moves `unittest` before torch, for `instance`

Reply:
It's correct to move `unittest` before `torch` because `unittest` is stdlib.